### PR TITLE
Userprofile

### PR DESCRIPTION
**Pull request by cy**

### CODE REVIEW

File: models/UserProfile.ts [hunk 1]
```diff
@@ -2,21 +2,39 @@ import { Schema, model } from 'mongoose';
 
 const Role = {
   type: String,
-  enum: ['Patient', 'Parent', 'Clinician', 'Researcher'],
+  enum: ['researcher', 'patient', 'clinician', /*'Advocacy Group'*/],
```
Comment (alex-wilmer) [inline models/UserProfile.ts]: why comment?

Comment (cy) [reply models/UserProfile.ts]: oh cuz Advocacy Group is in the spec and will be there eventually but there's no designs/icons for it yet

Comment (cy) [reply models/UserProfile.ts]: i guess uncommenting it here is fine, as long as it's not in the portal code it wouldn't show up. and shoot forgot to change patient to patient/caregiver. coming up

Comment (cy) [reply models/UserProfile.ts]: actually it's not letting the enums have spaces 💩

**Comment by alex-wilmer** [comment]
Is this in the wrong repo? Are we hardcoding kids-first stuff in overture/persona?

**Comment by cy** [comment]
hmmmm but where would these fields go then? config file for the schema maybe?

**Comment by cy** [comment]
going to merge this for now because need it for the demo today. we can talk about it more later.

**Comment by alex-wilmer** [comment]
Yeah a config file works

**Comment by alex-wilmer** [comment]
this uses mongo right? maybe we can add a config ui / config collection for this as well. config files might be annoying to maintain once more features are added

**Pull request merged**